### PR TITLE
Overhaul the front page introduction

### LINKED ISSUE #122: Make rotating screenshots larger
The screenshots on the main page are tiny! They are actually 500x300 but scaled down.

Here's how it would look with larger screenshots and a new header font:
oraweb

### LINKED ISSUE #241: Add Steam group to community page.
http://steamcommunity.com/groups/openra4life/
is the most well maintained group and should be deemed the official group.

### LINKED ISSUE #249: Suggestion: Connect the web pages
Hey, I noticed that there are alot sub pages and they are not connected enough in my opinion.
I suggest to add a link to gamereplays.org at the resource center
and also to add the resources center at the navigation bar of the main page.
This could boost value of these sub pages.

Can also found here:
https://github.com/OpenRA/OpenRA/issues/8904#issuecomment-127134061

### PR DESCRIPTION
**Pull request by pchote**
frontpage

Closes #122.
Closes #241.
Partially resolves #249.

### CODE REVIEW

**Comment by Mailaender** [comment]
The "Install Now" and "About" button is redundant to the menu bar and the stitching border looks terribly out of place. I suggest you highlight the existing ones instead.

**Comment by obrakmann** [comment]
Hmm, maybe with a red border around the box?
screenshot from 2016-03-27 17 08 57

File: content/style.css [hunk 1]
```diff
@@ -413,11 +478,17 @@ pre {
...
+    -webkit-border-radius: 10px;
+    -khtml-border-radius: 10px;
 	line-height: 0;
 }
 
 .about-modimage img {
 	border-radius: 8px;
+    -moz-border-radius: 8px;
+    -webkit-border-radius: 8px;
+    -khtml-border-radius: 8px;
```
Comment (Mailaender) [inline content/style.css]: Tabs instead of spaces.

File: content/style.css [hunk 2]
```diff
@@ -201,6 +226,48 @@ p.download, .newsDetails {
...
+.mainlinks #download:hover { background: url("/images/install-button-hover.svg") }
+.no-svg .mainlinks #download { background: url("/images/install-button-normal-fallback.png") }
+.no-svg .mainlinks #download:hover { background: url("/images/install-button-hover-fallback.png") }
+
+#newsbanner {
+    float:right;
+    font-size: 1em;
+	padding: 4px 10px 2px 10px;
+	margin-top:10px;
+    display:inline-block;
```
Comment (Mailaender) [inline content/style.css]: Mix of spaces and tabs.

File: content/style.css [hunk 3]
```diff
@@ -171,14 +186,24 @@ p.download, .newsDetails {
...
+#rightpanel p { margin: 10px;}
+
 #so-links {
-  margin-top:180px;
   text-align: center;
-  margin-bottom:0;
 }
 
 #so-links ul {
   border-radius: 4px;
```
Comment (Mailaender) [inline content/style.css]: This probably wants to be 10px as well then.

File: content/index.html [hunk 4]
```diff
@@ -1,33 +1,36 @@
...
 %>
   </div>
-  <div id="so-links">
-    Find us on your favorite social network:
-    <ul>
+  <div id="rightpanel">
+    <p>Classic strategy games, rebuilt for the modern era.</p>
+	<p>Open Source. Free. For all modern operating systems.</p>
+    <ul class="mainlinks">
+	  <li><a id="about" href="/about/" alt="About OpenRA" title="About OpenRA">About OpenRA</a></li>
```
Comment (Mailaender) [inline content/index.html]: Confusing indentions.

**Comment by pchote** [comment]
Discussion in IRC rejected the idea of a hero banner as unwanted duplication.  The screenshots should instead be moved to a media page, and the social links to the community page.

**Comment by x-a-n-a-x** [comment]
@pchote : is it what you would prefer? Moving screenshots to a media page and social links to the community page ? Or you said that reluctantly ?

I tend to think that that would be hiding social links from the direct sight of the visitor, cancelling the purpose of a quick click on these icons, imo. I know some of you are not fans of social networks but it's the way things are done these days (spreading news this way).

Also, I am not against a media page (in theory) but I do like the idea to keep rotating screenshots on the front page. If we remove all images from the front page, the visitor would have to click on Media to have an overview of the project. Again, we should provide our possible new players, a quick overview of the project at a glance.

And regarding https://github.com/OpenRA/OpenRAWeb/pull/281#issuecomment-202082491, I am not against it. I am just not found of the CSS style for the "About" and "Install Now" buttons.
Also, I think we should pick another font for "Welcome Back, Commander".

**Comment by pchote** [comment]
@x-a-n-a-x somewhat reluctantly, but I don't see much point in fighting against opinions ranging from dislike to "meh" at best.

**Comment by x-a-n-a-x** [comment]
In the meantime, couldn't we at least expand the social network DIV vertically, adding an extra row, to fit the other interesting icons:
- gamereplays
- itch.io
- steam group
- ...

social_icons

hmm ?

**Comment by obrakmann** [comment]
If you added a Youtube link you could have two symmetric rows of five icons each

**Comment by obrakmann** [comment]
How about with the extra `About` button removed, like this?

bc9b9688-f43e-11e5-9fd6-250cac7f3dc9

 The remaining button could maybe moved a bit down still.
bc9b9688-f43e-11e5-9fd6-250cac7f3dc9

**Comment by x-a-n-a-x** [comment]
Here is the itch.io icon
itchio2

**Comment by pchote** [comment]
@obrakmann can you try removing the github icon and going with a 3x3 layout? I suspect that will look better.

@x-a-n-a-x these social icons need to be svg format, with png fallbacks.

**Comment by x-a-n-a-x** [comment]
They have a white .svg icon here : https://itch.io/static/images/itchio-textless-white.svg
(but I don't think we should directly link this url from our frontpage)

**Comment by pchote** [comment]
I have amended the branch based on @obrakmann's mockup and the discussion in IRC.

If people are happy with this design then I will make svg icons for youtube and itchio and open a new pr.
screen shot 2016-05-13 at 18 45 07

**Comment by pchote** [comment]
Alternatively, if we trim 25px of height off the images we get:
screen shot 2016-05-13 at 18 56 13

**Pull request closed without merging**